src/dotty/tools/dotc/repl/REPL.scala

```diff
@@ -65,6 +65,9 @@ object REPL {
      */
     val cleanupCommands: List[String] = Nil
 
+    /** We also allow binding initial values */
```

Comment:
The description should not start with "We also allow ..." but just explains what bound values are and maybe give an example.